rust/src/mask/config/mod.rs

```diff
@@ -129,29 +128,30 @@ pub enum ModelType {
 }
 
 impl ModelType {
+    /// Gets the number of models to be aggregated at most for this model type.
     pub fn nb_models_max(&self) -> usize {
-        10usize.pow(*self as u8 as u32)
+        10_usize.pow(*self as u8 as u32)
     }
 }
 
 impl TryFrom<u8> for ModelType {
-    type Error = InvalidMaskConfig;
+    type Error = InvalidMaskConfigError;
 
-    /// Get the model type. Fails if the encoding is unknown.
     fn try_from(byte: u8) -> Result<Self, Self::Error> {
-        use ModelType::{M12, M3, M6, M9};
-        Ok(match byte {
-            3 => M3,
-            6 => M6,
-            9 => M9,
-            12 => M12,
-            _ => return Err(InvalidMaskConfig::ModelType),
-        })
+        match byte {
+            3 => Ok(ModelType::M3),
+            6 => Ok(ModelType::M6),
+            9 => Ok(ModelType::M9),
+            12 => Ok(ModelType::M12),
+            _ => Err(InvalidMaskConfigError::ModelType),
+        }
     }
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
-/// A mask configuration name. Consists of identifiers for its parts:
+/// A mask configuration.
```

Comment:
Not sure, but shouldn't we speak about "masking configuration" rather than "mask configuration"?

Reply:
not sure as well^^ i guess "masking configuration" is closer to the process and less focused on the actual mask. i'll change it accordingly. the english language can be strange.